samples/AspNetIntegration/BodyBindingHttpTrigger.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using FromBodyAttribute = Microsoft.Azure.Functions.Worker.Http.FromBodyAttribute;
```

Comment:
This wasn't needed for the base implementation for the non-ASP.NET Core version of HttpTrigger, but it does stand out here. Even with documentation, I think a number of folks will miss this and end up using the version from ASP.NET Core.

Reply:
Yes, this is present here to disambiguate the type reference as `Microsoft.AspNetCore.Mvc` is also being used. It is unfortunate, but there are a few things we can do to avoid confusion. We discussed some of those, but to capture here, we could also honor MVC attribute and leverage analyzers to help.